components/camera/config.go

```diff
@@ -4,17 +4,17 @@ import (
 	"github.com/pkg/errors"
 )
 
-// StreamType specifies what kind of image stream is coming from the camera.
-type StreamType string
+// ImageType specifies what kind of image stream is coming from the camera.
+type ImageType string
 
 // The allowed types of streams that can come from a VideoSource.
 const (
-	UnspecifiedStream = StreamType("")
-	ColorStream       = StreamType("color")
-	DepthStream       = StreamType("depth")
+	UnspecifiedStream = ImageType("")
+	ColorStream       = ImageType("color")
+	DepthStream       = ImageType("depth")
 )
 
 // NewUnsupportedStreamError is when the stream type is unknown.
-func NewUnsupportedStreamError(s StreamType) error {
+func NewUnsupportedStreamError(s ImageType) error {
```

Comment:
Rename this NewUnsupportedImageTypeError, as well as the text of the error

Reply:
Done